### PR TITLE
fix: modal style update

### PR DESCRIPTION
**Pull request by bertya**
related to ticket #544

### CODE REVIEW

File: packages/modal/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "pcln-modal",
-  "version": "2.1.0",
+  "version": "2.1.1",
```
Comment (jes708) [inline packages/modal/package.json]: Version updates can wait until the publishing process, as per the [Publishing guidelines](https://github.com/pricelinelabs/design-system/blob/master/CONTRIBUTING.md#publishing)

File: packages/modal/src/Modal.js [hunk 2]
```diff
@@ -43,7 +44,12 @@ const Overlay = styled(DialogOverlay)`
   ${props => `
     z-index: ${props.zindex || 100};
     font-family: ${props.theme.font};
+    line-height: ${props => props.theme.lineHeights.standard};
```
Comment (jes708) [inline packages/modal/src/Modal.js]: Should we be concerned about keeping this in sync with <Base /> from ThemeProvider?

https://github.com/pricelinelabs/design-system/blob/c2ad5c588944d5713bb6c61f32ce20a81368d31b/packages/core/src/ThemeProvider.js#L6-L14

Comment (bertya) [reply packages/modal/src/Modal.js]: personally, I feel it's fine since it's grabbing things from the theme,

Comment (jes708) [reply packages/modal/src/Modal.js]: Ok, that sounds fair to me

**Pull request closed without merging**